src/components/Table/Table.stories.tsx

```diff
@@ -243,3 +250,85 @@ export const AlignTableCellContentRight: StoryObj<Args> = {
     ),
   },
 };
+
+const SortableExample = () => {
+  const values = [
+    { col1: 'Value 1', col2: 'Value A' },
+    { col1: 'Value 3', col2: 'Value B' },
+    { col1: 'Value 2', col2: 'Value C' },
+    { col1: 'Value 4', col2: 'Value D' },
+  ];
+  const [sortDirection, setSortDirection] =
+    useState<SortDirectionsType>('default');
+  const [tableValues, setTableValues] = useState({ ...values });
+  const onSortClick = () => {
```

Comment:
Just curious—what was the reasoning for pulling the sort logic out of the component?

Reply:
The `Table` component never had sort logic to begin with, unless you're asking to add sorting logic to the `Table` component, in which case there would need to be context being thrown around and there is less value to be gained if the parent component can simply sort the data to begin with

Reply:
I guess I'm thinking of the `toggleSort` logic that was pulled out of `TableHeaderCell`, but it's strange how that logic updated the icon without hooking into anything else.

Reply:
Yea that was used for which icon to display only and had no actual sorting functionality hooking into rows or cells